test/fixtures/coverage/functional_test/tool/dev.dart

```diff
@@ -0,0 +1,9 @@
+library tool.dev;
+
+import 'package:dart_dev/dart_dev.dart' show dev, config;
+
+main(List<String> args) async {
+  config.test.functionalTests = ['functional'];
```

Comment:
I guess this is because of the requirement that functional tests currently must be run with the old test package... bummer